generated/symbols-index.ts

```diff
@@ -86,6 +88,8 @@ export default {
   "schottky_diode_vert": _schottky_diode_vert,
   "silicon_controlled_rectifier_horz": _silicon_controlled_rectifier_horz,
   "silicon_controlled_rectifier_vert": _silicon_controlled_rectifier_vert,
+  "text_paths_sample_horz": _text_paths_sample_horz,
+  "text_paths_sample_vert": _text_paths_sample_vert,
```

Comment:
we can use a snapshot test outside of the regular symbols definitions using https://github.com/tscircuit/bun-match-svg, that way this doesn't increase bundle size for symbols or get mixed in with electrical symbols